src/PriceOracle.sol

```diff
@@ -2,4 +2,22 @@
 pragma solidity ^0.8.4;
 import "@openzeppelin/contracts/access/Ownable.sol";
 
-contract PriceOracle is Ownable {}
+contract PriceOracle is Ownable {
+    uint256 public price;
+    uint256 public chequeValueDeduction;
+
+    function getPrice() external view returns (uint256, uint256) {
```

Comment:
Natspec comments in functions and contract

Reply:
done